walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -270,5 +361,55 @@ public void notEnoughRoom() {
 
     Assert.assertEquals(expected, actual);
   }
-}
 
+  @Test
+  public void optionalAttendeesOnlyWithRoom() {
```

Comment:
Unfortunate that you don't have access to our internal best practice guides. But keep in mind that the naming of these existing tests do not follow our best practice for testing.

> Each test should explicitly name not only the method being tested, but also the expected outcome and sometimes specific details of the state or input being tested.

For example, I would name these last 2 tests `query_onlyOptionalAttendees_returnsAvailableTime` `query_onlyOptionalAttendeesWithNoAvailableTime_returnsNoTimes`.
No need to change since you're just following existing convention in the code you're modifying.

Here's an external guide I found. https://qualitycoding.org/unit-test-naming/
The most important thing is the expected result and the condition of the test. When someone else makes changes to your code, if you didn't state the expected result, they can end up modifying your test *just to make the test pass* but the test may not be testing what it's meant to. Happens all the time.